packages/treasure/src/test-transactions.ts

```diff
@@ -105,6 +148,7 @@ export const failingSwapTestCases = [
     tokenOut: GFLY,
   }),
   createTestCase(EXACT_TOKENS_FOR_TOKENS, 'when amountIn is insufficient', {
+    // should these amounts be hard-coded as values lower than the filters in the test params above?
```

Comment:
Yeah - I think one of the _only_ places hardcoding makes sense is for one-off failing test cases.